package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "golf-athlete-api",
+  "version": "1.0.0",
+  "description": "RESTful API with CRUD endpoints for managing golf athletes",
+  "author": "Charlie Udom <charlie.udom@gmail.com>",
+  "license": "ISC",
+  "main": "server.js",
+  "scripts": {
+    "start": "node ./bin/server.js",
+    "devstart": "nodemon ./bin/server.js",
+    "stylint": "standard \\\"src/**/*.js\\\" | snazzy",
+    "stylint:fix": "standard --fix",
+    "test": "npm run stylint && npm run test:coverage",
+    "test:coverage": "nyc --check-coverage --lines 95 --functions 95 --branches 95 npm run test:all",
+    "test:all": "mocha ./test/**/*.spec.js --timeout 5000 --exit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cudom/node-backend-test"
+  },
+  "dependencies": {
+    "acorn": "^6.1.1",
```

Comment:
why u need `acorn`?

Reply:
When I installed one of the other dependencies, I got an error about acorn also being required. I installed to move forward in the project and haven't had a chance to look further into it yet. I do think that unnecessary packages should not be included.